app/assets/stylesheets/layouts/starrydark.scss

```diff
@@ -1,157 +1,287 @@
 @import 'variables';
+
+// TODO: convert to more proper variables
```

Comment:
Can we extract this to a GitHub issue? I expect it'll get buried and clutter just one of the files, otherwise.

Reply:
...I think I actually fixed that did I not remove to comment apparently not